package.json

```diff
@@ -39,38 +39,38 @@
     "@types/marked": "0.0.28",
     "@types/mathjax": "0.0.31",
     "@types/mocha": "^2.2.32",
-    "@types/node": "0.0.1",
+    "@types/node": "7.0.5",
     "@types/sanitize-html": "^1.13.31",
     "@types/semver": "^5.3.30",
     "chai": "^3.5.0",
-    "concurrently": "^2.0.0",
-    "css-loader": "^0.23.1",
+    "concurrently": "^3.4.0",
+    "css-loader": "^0.26.2",
     "expect.js": "^0.3.1",
-    "file-loader": "^0.8.5",
+    "file-loader": "^0.10.1",
     "font-awesome": "^4.6.3",
-    "fs-extra": "^0.26.4",
-    "istanbul-instrumenter-loader": "^0.1.3",
+    "fs-extra": "^2.0.0",
+    "istanbul-instrumenter-loader": "^2.0.0",
     "json-loader": "^0.5.4",
     "json-to-html": "^0.1.2",
-    "karma": "^0.13.19",
-    "karma-chrome-launcher": "^0.2.2",
-    "karma-coverage": "^0.5.3",
-    "karma-firefox-launcher": "^0.1.7",
-    "karma-ie-launcher": "^0.2.0",
-    "karma-mocha": "^0.2.1",
-    "karma-mocha-reporter": "^1.1.5",
+    "karma": "^1.5.0",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-coverage": "^1.1.1",
+    "karma-firefox-launcher": "^1.0.1",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.2",
     "karma-remap-coverage": "^0.1.1",
     "karma-sourcemap-loader": "^0.3.7",
-    "mocha": "^2.3.4",
+    "mocha": "^3.2.0",
     "raw-loader": "^0.5.1",
     "rimraf": "^2.5.0",
     "simulate-event": "^1.2.0",
     "style-loader": "^0.13.0",
     "typedoc": "^0.5.0",
-    "typescript": "~2.1.6",
+    "typescript": "~2.2.1",
     "url-loader": "^0.5.7",
-    "watch": "^0.17.1",
-    "webpack": "^1.12.11"
+    "watch": "^1.0.2",
+    "webpack": "^2.2.1"
```

Comment:
(because we have a custom webpack [plugin](https://github.com/jupyterlab/extension-builder/blob/master/src/plugin.ts))